### PR TITLE
Invibes Bid Adapter: support for custom domainId

### PR DESCRIPTION
**Pull request by rcheptanariu**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added support for additional bid param **domainId** and also custom extensions for placement ids

- contact email of the adapter’s maintainer
system_operations@invibes.com

- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo
https://github.com/prebid/prebid.github.io/pull/3440

### CODE REVIEW

File: modules/invibesBidAdapter.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ Connect to Invibes for bids.
                    {
                        bidder: 'invibes',
                        params: {
-                           placementId: '12345'
+                           placementId: '12345',
+						   networkId: 1001
```
Comment (smenzer) [inline modules/invibesBidAdapter.md:24]: should this be `domainId`? I don't see `networkId` listed anywhere else in your change set

also, can you fix the spacing please

File: test/spec/modules/invibesBidAdapter_spec.js [hunk 2]
```diff
@@ -158,6 +158,125 @@ describe('invibesBidAdapter:', function () {
       expect(request.method).to.equal('GET');
     });
 
+    it('sends bid request to custom endpoint via GET', function () {
+      const request = spec.buildRequests([{
+        bidId: 'b1',
+        bidder: BIDDER_CODE,
+        params: {
+          placementId: 'placement',
+	  	  customEndpoint: 'sub.domain.com/Bid/VideoAdContent'
```
Comment (smenzer) [inline test/spec/modules/invibesBidAdapter_spec.js:167]: spacing

File: test/spec/modules/invibesBidAdapter_spec.js [hunk 3]
```diff
@@ -158,6 +158,125 @@ describe('invibesBidAdapter:', function () {
...
+      expect(request.url).to.equal(ENDPOINT);
+      expect(request.method).to.equal('GET');
+    });
+
+    it('sends bid request to default endpoint when null placement', function () {
+      const request = spec.buildRequests([{
+        bidId: 'b1',
+        bidder: BIDDER_CODE,
+        params: {
+		  placementId: null
```
Comment (smenzer) [inline test/spec/modules/invibesBidAdapter_spec.js:192]: spacing

File: test/spec/modules/invibesBidAdapter_spec.js [hunk 4]
```diff
@@ -158,6 +158,125 @@ describe('invibesBidAdapter:', function () {
...
+      expect(request.method).to.equal('GET');
+    });
+
+    it('sends bid request to network id endpoint 1 via GET', function () {
+      const request = spec.buildRequests([{
+        bidId: 'b1',
+        bidder: BIDDER_CODE,
+        params: {
+          placementId: 'placement',
+	  	  domainId: 1001
```
Comment (smenzer) [inline test/spec/modules/invibesBidAdapter_spec.js:219]: spacing

File: test/spec/modules/invibesBidAdapter_spec.js [hunk 5]
```diff
@@ -158,6 +158,125 @@ describe('invibesBidAdapter:', function () {
...
+      expect(request.method).to.equal('GET');
+    });
+
+    it('sends bid request to network id endpoint 2 via GET', function () {
+      const request = spec.buildRequests([{
+        bidId: 'b1',
+        bidder: BIDDER_CODE,
+        params: {
+          placementId: 'placement',
+	  	  domainId: 1002
```
Comment (smenzer) [inline test/spec/modules/invibesBidAdapter_spec.js:233]: spacing

**Review by smenzer — CHANGES_REQUESTED** [review]
just a few style issues and a question, otherwise looks good. once addressed this can be merged

File: modules/invibesBidAdapter.md (hunk 1 above)
Comment (rcheptanariu) [reply modules/invibesBidAdapter.md]: yes, it should have been domainId !

**Comment by rcheptanariu** [comment]
@smenzer All issues have been addressed. Please review again at your convenience ! :)

**Review by smenzer — APPROVED** [review]
LGTM

**Pull request merged**